rest.rst

```diff
@@ -285,68 +284,6 @@ Selecting Devices to Test On
 
 To select devices to test on, go to testmunk.com and navigate to ``Account Settings`` > ``REST API``. Or you can also set the devices the moment you create a testrun using the `Create a new testrun`_ endpoint.
 
-Start an existing testrun
```

Comment:
@mposchen Actually, it's possible to start an existing testrun. I've just found how. First you need to create new testrun and set `autoStart=false`: 
![image](https://cloud.githubusercontent.com/assets/7532782/7713684/a209c3a0-fe79-11e4-8e16-f33225fae798.png)

And then, you need to have the "id" (from previous call) and you can use `/run` to start it.

There are not many use cases for this. Nevertheless, we can leave it. @mposchen How do you think?